### DISCUSSION TITLE
Remove `OS.Month` and `OS.Weekday` enums since there are `Time`'s copies

### DISCUSSION THREAD

**Discussion by dalexeev** [godotengine/godot-proposals#5627] [Engine Core]
[OS.Month](https://docs.godotengine.org/en/latest/classes/class_os.html#enum-os-month)
[OS.Weekday](https://docs.godotengine.org/en/latest/classes/class_os.html#enum-os-weekday)

[Time.Month](https://docs.godotengine.org/en/latest/classes/class_time.html#enum-time-month)
[Time.Weekday](https://docs.godotengine.org/en/latest/classes/class_time.html#enum-time-weekday)

**Comment by Calinou** [comment]
For reference, the Time singleton was added in https://github.com/godotengine/godot/pull/49123.

cc @aaronfranke

**Comment by aaronfranke** [comment]
Opened a PR for this: https://github.com/godotengine/godot/pull/67694
